meson_options.txt

```diff
@@ -7,8 +7,8 @@ option('enable-initd', type : 'feature', value : 'disabled', description : 'Enab
 option('enable-python', type : 'feature', value : 'auto', description : 'Install the pygstc library')
 
 # String options
-option('with-gstd-runstatedir', type : 'string', value : '${prefix}/var/run/gstd', description : 'Specify the location of the gstd\'s PID file')
-option('with-gstd-logstatedir', type : 'string', value : '${prefix}/var/log/gstd', description : 'Specify the location of gstd\'s log files')
+option('with-gstd-runstatedir', type : 'string', value : '/var/run/gstd', description : 'Specify the location of the gstd\'s PID file')
```

Comment:
I actually think meson was in the right here. The default should be [${localstatedir}](https://mesonbuild.com/Builtin-options.html#directories)/run/gstd and [${localstatedir}](https://mesonbuild.com/Builtin-options.html#directories)/log/gstd. Having auto tools directly install into `/var/` was a bit outreachy of our part.

Reply:
${localstatedir} is not defined here.

According to the file-system standard both options are wrong (the meson default `/usr/var/...` and the autotools hardwired path `/var/...`). 

- The meson default `/usr/var` (with `--prefix=/usr`) is not part of the [filesystem standard](https://www.pathname.com/fhs/pub/fhs-2.3.html#THEUSRHIERARCHY) and so applications shouldn't create it (lintian error `non-standard-dir-in-usr`).  
- `/var/log/gstd` is actually OK by the [filesystem standard](https://www.pathname.com/fhs/pub/fhs-2.3.html#VARLOGLOGFILESANDDIRECTORIES).
- `/var/run/gstd` is forbidden because according to the standard the `/var/run` sub-folder structure can only be modified during boot because it may be a temporary filesystem (lintian error `dir-or-file-in-var-run`).

I think that the best of the two options is the one without `${prefix}` because the lintian rule we are breaking in that case seems less critical as It only affects systems where `/var/run` is a temporary fs. It can be solved in the future by adding a boot script that creates and gives permissions to these folders.